docs/modeling-your-data/modeling-your-data-with-dbt/migration-guides/unified/index.md

```diff
@@ -419,3 +428,38 @@ rename to snowplow_unified_views;
 </TabItem>
 </Tabs>
 </details>
+
+#### 2. Updates to the users table
+
+* New column: `absolute_time_in_s`
+
+<details>
+  <summary>SQL scripts</summary>
+<Tabs groupId="warehouse" queryString>
+<TabItem value="snowflake" label="snowflake" default>
+
+```sql
+ALTER TABLE	(your_schema)_DERIVED.SNOWPLOW_UNIFIED_USERS ADD absolute_time_in_s NUMBER(34,0);
+```
+
```

Comment:
Sorry, they don't actually need to do anything for new columns, dbt manages that all for them automatically thankfully, it's just type changes

Reply:
removed the new column specs